src/navigation/Routes.android.js

```diff
@@ -361,7 +361,11 @@ function AuthNavigator() {
 }
 
 const AppContainerWithAnalytics = React.forwardRef((props, ref) => (
-  <NavigationContainer onStateChange={onNavigationStateChange} ref={ref}>
+  <NavigationContainer
+    onReady={props?.onReady}
```

Comment:
I think it doesn't need to be optional you're right. I was just trying to make it consumer friendly by default but this couple of lines won't be used in any other way

Reply:
I think it doesn't need to be optional you're right. I was just trying to make it consumer friendly by default but this couple of lines won't be used in any other way

Reply:
Another thing I had in mind was that it was JS, so you won't be type checked. That's mostly why I sued optional chaining, in case someone removes sentry x navigation integration it won't crash